tests.sh

```diff
@@ -438,7 +438,7 @@ run_mdtest(){
                 -x CPATH -x PATH -x LD_LIBRARY_PATH
                 -x CRT_PHY_ADDR_STR -x OFI_DOMAIN -x OFI_INTERFACE
                 -x FI_MR_CACHE_MAX_COUNT -x FI_UNIVERSE_SIZE
-                -x FI_VERBS_PREFER_XRC
+                -x FI_VERBS_PREFER_XRC -x FI_OFI_RXM_USE_SRX
```

Comment:
I am wondering if we need to export this new variable for self tests at line 394

Also, not sure if we need it for starting the agent and creating containers at lines 198 and 256
`export FI_OFI_RXM_USE_SRX=${FI_OFI_RXM_USE_SRX};`